src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionsState.cs

```diff
@@ -409,7 +436,6 @@ private void CancelSingleConnection(SubscriptionConnection connection, Subscript
             {
                 connection.ConnectionException = ex;
                 connection.CancellationTokenSource.Cancel();
-                RegisterRejectedConnection(connection, ex);
```

Comment:
this is a change, why it was removed? can you explain?

Reply:
this was adding we the subscription to both lists `RecentConnections` & `RejectedConnection`

In general now we should log the subscription properly:
-  `RejectedConnection` -  holds the connections that could not subscribe on connection (for example new TakeOver when there is active TakeOver)
- `RecentConnections` - holds all connections that were active (no matter if got exception or closed)
- `PendingConnections` - holds the connections that are waiting for free spot (`WaitForFree`)